lib/network/http2_network_test.go

```diff
@@ -0,0 +1,179 @@
+package sebaknetwork
```

Comment:
This test file is the most important one.
Main Test code is [here](https://github.com/bosnet/sebak/pull/104/files#diff-e3aeb635ada062f1fadb637d5a0f6232R101)